test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -99,37 +90,35 @@ describe('Request to go live: organisation name page', () => {
       cy.get('#request-to-go-live-organisation-name-form').should('exist')
       cy.get('input#request-to-go-live-organisation-name-input').should('exist')
 
+      cy.get('input#request-to-go-live-organisation-name-input').should('be.empty')
+
       cy.get('input#request-to-go-live-organisation-name-input').type(organisationName)
       cy.get('input#request-to-go-live-organisation-name-input').should('have.value', organisationName)
 
       cy.get('#request-to-go-live-organisation-name-form > button').should('exist')
       cy.get('#request-to-go-live-organisation-name-form > button').should('contain', 'Continue')
-    })
-
-    it('should show empty input box if organisation name is not pre-filled', () => {
-      const requestToGoLivePageOrganisationNameUrl = `/service/${serviceExternalId}/request-to-go-live/organisation-name`
-      cy.visit(requestToGoLivePageOrganisationNameUrl)
-
-      cy.get('h1').should('contain', 'What is the name of your organisation?')
-
-      cy.get('#request-to-go-live-current-step').should('exist')
+      cy.get('#request-to-go-live-organisation-name-form > button').click()
 
-      cy.get('#request-to-go-live-organisation-name-form').should('exist')
-      cy.get('input#request-to-go-live-organisation-name-input').should('exist')
-      cy.get('input#request-to-go-live-organisation-name-input').should('be.empty')
+      cy.location().should((location) => {
+        expect(location.pathname).to.eq(`/service/${serviceExternalId}/request-to-go-live/choose-how-to-process-payments`)
+      })
     })
   })
 
-  describe('Service has correct go live stage and organisation name is pre-filled', () => {
+  describe('Service has NOT_STARTED go live stage and organisation name is pre-filled', () => {
+    const serviceRolesBefore = utils.buildServiceRoleForMerchantDetailsField({ name: organisationName })
```

Comment:
Would be good to explicitly set the `current_go_live_stage` in the stub here, as currently we are relying on the fixture default of `NOT_STARTED` which we can't rely on to not change, maybe make this a parameter of the method.